### PR TITLE
Extra Emojis for cache icons / fix SMILEY_HEART Emoji

### LINKED ISSUE #16371: Additional icons
### Tell us your idea!

There are a few cache icons that I would like to have in the app. Obviously I can use others in the meantime, but it would be nice to not have to make a mental mapping 

### What solution would you suggest?

- 💰 or 🪙  (helpful to indicate if a cache requires paid entry) 
- ~~✅ and ❎ (helpful for solved puzzles, etc)~~ (Actually these are already there)
- other cache types (e.g. a mystery cache that is really more like a multicache)

### Is there an alternative?

_No response_

### Additional Information

As a programmer myself, I'm happy to try add these (pointers to places in code would be appreciated). But the readme says to file a ticket rather than begin hacking.

If I do so I might look to include some other icons like 🤷 👪 🐕 for others' benefit. But for now the ones I want are listed above

### PR DESCRIPTION
**Pull request by ianfhunter**
### Added New Emoji
🪙 💰- indicating caches requiring money (e.g. access fee, parking)
🛶 ⚓ ✈️ 🧳 🚀 🛸 🪂 🐴 - indicators for travel 
🧩 🤷 - additional icons for puzzle caches
🐶 🐱 - Indicators for dog/cat-friendly caches

### Fixed Incorrect Emoji
💙 was in the emoji list twice - one of them was meant to be SMILEY_HEART (Not sure which was this one so I chose 😍 )

## Related issues

Fixes #16371  - apart from other cache icons (e.g. marking a mystery cache as a traditional). This is more involved than I thought.

### CODE REVIEW

File: main/src/main/java/cgeo/geocaching/utils/EmojiUtils.java [hunk 1]
```diff
@@ -133,7 +136,8 @@ public class EmojiUtils {
                     /* smileys */       SMILEY_LIKE, SMILEY_LOVE, 0x1f641, 0x1f621, SMILEY_MONOCLE,
                     /* silhouettes */   0x1f47b, 0x1f464, 0x1f465, 0x1f5e3,
                     /* people */        0x1f466, 0x1f467, 0x1f468, 0x1f469, 0x1f474, 0x1f475, 0x1f46a,
-                    /* hands */         0x1f44d, 0x1f44e, 0x1f4aa, 0x270d
+                    /* hands */         0x1f937, 0x1f44d, 0x1f44e, 0x1f4aa, 0x270d,
+                    /* companions */    0x1f434, 0x1f408, 0x1F415, 0x1f415
```
Comment (ianfhunter) [inline main/src/main/java/cgeo/geocaching/utils/EmojiUtils.java:140]:
```suggestion
                    /* companions */    0x1f434, 0x1f408, 0x1f415
```

File: main/src/main/java/cgeo/geocaching/utils/EmojiUtils.java [hunk 2]
```diff
@@ -122,9 +123,11 @@ public class EmojiUtils {
                     /* person-sport */  0x26f7, 0x1f3c4, 0x1f6a3, 0x1f3ca, 0x1f6b4,
                     /* p.-activity */   0x1f6b5, 0x1f9d7,
                     /* person-role */   0x1f575,
-                    /* sport */         0x26bd, 0x1f94e, 0x1f3c0, 0x1f3c8, 0x1f93f, 0x1f3bf, 0x1f3af, 0x1f9ff,
+                    /* sport */         0x26bd, 0x1f94e, 0x1f3c0, 0x1f3c8, 0x1f93f, 0x1f3bf, 0x1F6F6, 0x2693, 0x1f3af, 0x1f9ff,
```
Comment (ianfhunter) [inline main/src/main/java/cgeo/geocaching/utils/EmojiUtils.java:126]:
```suggestion
                    /* sport */         0x26bd, 0x1f94e, 0x1f3c0, 0x1f3c8, 0x1f93f, 0x1f3bf, 0x1f6f6, 0x2693, 0x1f3af, 0x1f9ff,
```

File: main/src/main/java/cgeo/geocaching/utils/EmojiUtils.java [hunk 3]
```diff
@@ -122,9 +123,11 @@ public class EmojiUtils {
...
                     /* tool */          0x1fa9c, 0x1f3a3, 0x1f6e0, 0x1fa9b, 0x1f512, 0x1f511, 0x1f50b, 0x1f9f2,
                     /* science */       0x2697, 0x1f9ea, 0x1f52c,
+                    /* games */         0x1f9e9,
+                    /* travel */        0x2708, 0x1f9f3, 0x1F680, 0x1F6F8, 0x1FA82,
```
Comment (ianfhunter) [inline main/src/main/java/cgeo/geocaching/utils/EmojiUtils.java:130]:
```suggestion
                    /* travel */        0x2708, 0x1f9f3, 0x1f680, 0x1f6f8, 0x1fa82,
```

**Comment by bekuno** [comment]
Ok to test

**Comment by moving-bits** [comment]
Thanks @ianfhunter for your contribution, very welcomed!

**Comment by moving-bits** [comment]
> 💙 was in the emoji list twice - one of them was meant to be SMILEY_HEART (Not sure which was this one so I chose 😍 )

Thanks for fixing this. 

The smiley you selected as replacement was actually the correct one, used some time ago, as you can see in the commit history for the chosen line. This can be reached in AndroidStudio by selecting a couple of lines, opening the context menu, then selecting "Git" => "Show History for selection".

**Pull request merged**